build.yaml

```diff
@@ -30,4 +34,8 @@ global_options:
     options:
       environment:
         PRE_NULL_SAFETY_SERVER_URL: https://v1.api.dartpad.dev/
-        NULL_SAFETY_SERVER_URL: https://nullsafety.api.dartpad.dev/
\ No newline at end of file
+        NULL_SAFETY_SERVER_URL: https://nullsafety.api.dartpad.dev/
+        STABLE_SERVER_URL: https://nullsafety.api.dartpad.dev/
+        BETA_SERVER_URL: https://beta.api.dartpad.dev/
+        DEV_SERVER_URL: https://dev.api.dartpad.dev/
+        OLD_SERVER_URL: https://old.api.dartpad.dev/
```

Comment:
nit: newline

Reply:
Haha yeah, I think I've gotten so used to autoformatters, I don't even think about carefully formatting a file any more. I'm sure I could install a VS Code extension for formatting YAML. Prettier, maybe.